src/lighteval/tasks/multilingual/tasks.py

```diff
@@ -3903,3 +3906,250 @@
         *hindi_boolq_tasks,
     ]
 )
+
+# ------------------------------- Translation Tasks ------------------------------- #
+flores_200_languages = [
+    "ace_Arab",
```

Comment:
Different scripts won't really work, because of how template is constructed :/.
We must choose one of the scripts and comment out the rest....

Reply:
might be more interesting to add the script to the Language enum and use it in the template functions I think

Reply:
I can take a look on Mon

Reply:
That would be the "correct" solution, yes. If you have time it would be great, but I think we are fine with just having one default script as of right now.

Reply:
Ok, then merging the simplest and I'll open an issue for adding script in the enum